src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -2082,6 +2127,7 @@ public Collection<Class<? extends LifecycleComponent>> getGuiceServiceClasses()
 
         final List<Class<? extends LifecycleComponent>> services = new ArrayList<>(1);
         services.add(GuiceHolder.class);
+        log.info("Guice service classes loaded");
```

Comment:
Do we want to change this to something lower like debug?

Reply:
Good catch. I will lower it to debug.